### PR TITLE
Sandbox blog

### PR DESCRIPTION
**Pull request by strickvl**
## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Describe changes
Briefly describe the changes you have introduced.

### CODE REVIEW

File: _posts/2023-05-31-sandbox-launch.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Simplifying MLOps Platform Deployment
+
+The goal of ZenML is to give ML/MLOps engineers the ability to pick and choose their preferred infrastructure and tooling to build a platform that fulfills their company's needs. However, deploying ZenML and a rudimentary MLOps platform can be time-consuming for new users. The ZenML Sandbox bridges this gap by providing a one-click deployment platform for a pre-built ephemeral MLOps stack, simplifying the deployment process.
+
+Users can sign up with Google and start a demo sandbox. After a few minutes, they are given credentials for Kubeflow, Minio, MLflow, and ZenML. Users can then use commands like `zenml connect` and `zenml stack set` to set their stacks and `python run.py` to run the pipelines. The sandbox is deleted after 6 hours, and users can choose from a repository of pre-built pipelines to run.
```
Comment (htahir1) [inline _posts/2023-05-31-sandbox-launch.md:25]:
```suggestion
Users can sign up with Google and start a demo sandbox. After a few minutes, they are given credentials for Kubeflow, Minio, MLflow, and ZenML. Users can then use commands like `zenml connect` and `zenml stack set` to set their stacks and `python run.py` to run the pipelines. The sandbox is deleted after 8 hours, and users can choose from a repository of pre-built pipelines to run.
```

**Comment by strickvl** [comment]
@htahir1 updated a bunch. Needs more screenshots to spruce it up, and maybe an extra paragraph motivating someone to use it in a specific sense (vs the general reasons given already)

File: _posts/2023-05-31-mlops-platform-sandbox.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+thumbnail: /assets/posts/sandbox/sandbox-launch-small.png
+image:
+  path: /assets/posts/sandbox/sandbox-launch.png
+---
+
+**Last updated:** May 31, 2023
+
+![Screenshots of the MLOps Platform Sandbox](/assets/posts/sandbox/sandbox-launch.png)
+
+We are excited to launch the MLOps Platform Sandbox, a one-click deployment platform for an ephemeral MLOps stack that you can use to run production-ready MLOps pipelines in the cloud. The MLOps Platform Sandbox allows you to create a sandbox with a stack made up of ZenML, Kubeflow, MLflow, and Minio Bucket. The Sandbox comes with pre-built example pipelines that you can run and try out. It provides a seamless experience for you to experiment with these tools without worrying about infrastructure setup and management.
```
Comment (htahir1) [inline _posts/2023-05-31-mlops-platform-sandbox.md:19]:
```suggestion
We are excited to launch the MLOps Platform Sandbox, a one-click deployment platform for an ephemeral MLOps stack that you can use to run production-ready MLOps pipelines in the cloud. The MLOps Platform Sandbox allows you to create a sandbox with a stack made up of [ZenML](https://zenml.io/), [Kubeflow](https://www.kubeflow.org/), [MLflow](https://mlflow.org/), and [Minio ](https://min.io/) Bucket. The Sandbox comes with pre-built example pipelines that you can run and try out. It provides a seamless experience for you to experiment with these tools without worrying about infrastructure setup and management.
```

File: _posts/2023-05-31-mlops-platform-sandbox.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## How to Use the Sandbox
+
+The Sandbox provides you with pre-built pipelines that you can easily re-run or modify to suit your needs. If you want to run your own custom code inside your sandbox, there are a few more steps you'll have to do, all of which is [described in our dedicated documentation page](https://docs.zenml.io/user-guide/advanced-guide/sandbox).
+
+To list the available pre-built pipelines with which your sandbox comes, simply use the command `zenml pipeline list`. This will display a list of pipelines, including their build IDs, which you can use to run a specific pipeline.
+
+To run a pipeline with a build, use the command `zenml pipeline run <PIPELINE_NAME> --build_id <BUILD_ID>`. For example, to run the `langchain_pipeline`, you would enter `zenml pipeline run langchain_pipeline --build_id <BUILD_ID>`, replacing `<BUILD_ID>` with the appropriate build ID from the list.
```
Comment (htahir1) [inline _posts/2023-05-31-mlops-platform-sandbox.md:39]: The langchain pipeline should be replaced with the mlflow example

File: _posts/2023-05-31-mlops-platform-sandbox.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+## What To Do After Your Sandbox Expires
+
+Once you have experimented with the MLOps Platform Sandbox and gained a better understanding of how the ZenML frameworks work, you may want to deploy your own MLOps stack tailored to your specific needs. To help you with this process, ZenML offers [Stack Recipes](https://github.com/zenml-io/mlops-stacks), which provide a starting point for deploying various MLOps stacks on different cloud providers and with different components.
+
+[Stack Recipes](https://github.com/zenml-io/mlops-stacks) are designed to be customizable, allowing you to easily swap out components and infrastructure based on your company's requirements. You can replace Kubeflow with Vertex AI Pipelines or Sagemaker Pipelines, use S3 or GCS storage instead of Minio, and choose Weights and Biases or Neptune over MLflow.
```
Comment (htahir1) [inline _posts/2023-05-31-mlops-platform-sandbox.md:47]: Maybe link to the docs instead?

File: _posts/2023-05-31-mlops-platform-sandbox.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+To get started with deploying your own MLOps stack, visit the [ZenML Stack Recipes](https://github.com/zenml-io/mlops-stacks) repository along with [our documentation page](https://docs.zenml.io/platform-guide/set-up-your-mlops-platform/deploy-and-set-up-a-cloud-stack/deploy-a-stack-post-sandbox) and follow the instructions provided for your desired stack configuration. This will enable you to build a robust, production-ready MLOps platform tailored to your specific needs, while leveraging the knowledge and experience gained from using the MLOps Platform Sandbox.
```
Comment (htahir1) [inline _posts/2023-05-31-mlops-platform-sandbox.md:49]: Maybe add a CTA to the form to extend the sandbox which is: https://zenml.io/extend-sandbox.

File: _posts/2023-05-31-mlops-platform-sandbox.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+## Why ZenML On Kubeflow / MLflow / Minio?
+
+IMAGE OF STACK GOES HERE
+
+Using ZenML with Kubeflow, MLflow, and Minio is a representative stack as it includes a production-ready orchestrator, an object storage component for data versioning, and a popular experiment tracking tool in machine learning. ZenML's stack recipe system is designed to allow you to easily swap out components and infrastructure based on your company's needs.
+
+## How to Use the Sandbox
+
+The Sandbox provides you with pre-built pipelines that you can easily re-run or modify to suit your needs. If you want to run your own custom code inside your sandbox, there are a few more steps you'll have to do, all of which is [described in our dedicated documentation page](https://docs.zenml.io/user-guide/advanced-guide/sandbox).
```
Comment (htahir1) [inline _posts/2023-05-31-mlops-platform-sandbox.md:35]: Maybe link to the corresping docs page here

**Review by htahir1 — CHANGES_REQUESTED** [review]
Left some comments but overall looking food!

File: _posts/2023-05-31-mlops-platform-sandbox.md (hunk 5 above)
Comment (strickvl) [reply _posts/2023-05-31-mlops-platform-sandbox.md:49]: that link gives me a 404. Also what form are you talking about?

Comment (htahir1) [reply _posts/2023-05-31-mlops-platform-sandbox.md:49]: This link will be fixed soon, its a form that lets people ask for an extended mlops platform sandbox

Comment (strickvl) [reply _posts/2023-05-31-mlops-platform-sandbox.md:49]: got it. I'll update the relevant part of docs as well? and the FAQ?

Comment (htahir1) [reply _posts/2023-05-31-mlops-platform-sandbox.md:49]: @bcdurak already added a card in the docs that I wrote, for the FAQ ill do it on the website directly

File: _posts/2023-05-31-mlops-platform-sandbox.md (hunk 6 above)
Comment (htahir1) [reply _posts/2023-05-31-mlops-platform-sandbox.md:35]: You're right

File: _posts/2023-05-31-mlops-platform-sandbox.md (hunk 3 above)
Comment (htahir1) [reply _posts/2023-05-31-mlops-platform-sandbox.md:39]: @strickvl Why did you resolve this when it isnt done? :-)

Comment (strickvl) [reply _posts/2023-05-31-mlops-platform-sandbox.md]: done but not pushed...

**Pull request merged**